e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -154,7 +154,7 @@ func InitializeMetricbeatScenarios(ctx *godog.ScenarioContext) {
 	ctx.BeforeScenario(func(p *messages.Pickle) {
 		log.Trace("Before Metricbeat scenario...")
 
-		tx = apm.DefaultTracer.StartTransaction(p.GetName(), "test.scenario")
+		tx = apme2e.StartTransaction(p.GetName(), "test.scenario")
```

Comment:
The import in this file is not present:

```go
apme2e "github.com/elastic/e2e-testing/internal"
```

Reply:
OTOH, we are planning to remove metricbeat test suite in #1520 (PR #1502), maybe it's not needed to be updated

Reply:
Sure, that's fine :)